### DISCUSSION TITLE
Conditional + Numeric filter shows non-conditionals

### DISCUSSION THREAD

**Discussion by hickman-santini** [Metaculus/metaculus#967] [Ideas]
There are conditional questions where the _children_ are numeric. That's what should show here. Fwiw pretty confident this was working earlier this week.

![image](https://github.com/user-attachments/assets/d379697f-f0b0-4cb3-b605-965ab41d14ed)

**Comment by SylvainChevalier** [comment]
I think the current behaviour is intended. All the "question type" filters are ORed together, not ANDed, otherwise e.g. "numeric"+"date" would return nothing. "Numeric" means single numeric questions, "conditional" means any conditional pair, etc.

I don't think "numeric"+"conditional" should return conditionals with numeric children, because then what happens if you also select "multiple choice"?

I see clean ways to do this, including:
1. Have a "type" filter (single, group, conditional), then a "subtype" filter (binary, numeric, date, multiple choice, notebook).
2. Split the groups and conditionals into: binary group, numeric group, etc. And maybe have a "any group" and a "any conditional" shortcut.
3. Do what e.g. notion does: keep the normal filter simple (or make it even simpler), but have an advanced filter where you select a property, then an operator, then a value. This is fully general and would also work for dates, statuses, etc.
![image](https://github.com/user-attachments/assets/c7d5c16b-b1a6-49c7-aeee-71f5947fe69b)


I think you can tell I prefer 3. 1. and 2. work, but at the cost of extra buttons or sections, and confusing names, which imo makes the filter menu way worse.
